packages/components/checkbox/src/CheckboxInput.styles.ts

```diff
@@ -3,10 +3,10 @@ import { cva, VariantProps } from 'class-variance-authority'
 
 export const checkboxInputStyles = cva(
   [
-    'h-sz-20 w-sz-20 border-md peer my-[var(--sz-1)] shrink-0 items-center justify-center self-baseline rounded-sm bg-transparent',
```

Comment:
@kikoruiz I removed some stuff that I thought it wasn't necessary. Can you double check please ? 🙏🏻

Reply:
I did't work on this component before but I guess the `peer` classname is not needed by any other sibling element, right?

Reply:
I checked on the main branch and `peer` was no longer needed :)